src/core/LinearExtrudeNode.cc

```diff
@@ -89,7 +89,11 @@ static std::shared_ptr<AbstractNode> builtin_linear_extrude(const ModuleInstanti
   }
 
   if (parameters["height"].isDefined()) {
-    parameters["height"].getFiniteDouble(node->height);
+    if(!parameters["height"].getVec3(node->height[0], node->height[1], node->height[2], 0.0)) {
```

Comment:
I wonder if we should name this parameter differently than "height" now that we support a vector.